resources/dicts/thoughts/alive/general.json

```diff
@@ -51,87 +49,1424 @@
+            "Is helping to reinforce the camp walls",
+            "Is patching up a hole in the camp wall",
+            "Is helping to reinforce the walls of the elders' den",
+            "Fortifies the camp walls for the fourth time this week",
+            "Feels at peace",
+            "Keeps an eye on some rowdy kits"
+        ],
+        "main_trait_constraint": [
+            "careful"
+        ]
+    },
+    {
+        "id": "charismatic_gen",
+        "thoughts": [
+            "Is smiling warmly at {PRONOUN/m_c/poss} Clanmates",
+            "Is making everyone laugh with a very funny comment",
+            "Always manages to turn the mood into a lighter one",
+            "Is laughing with friends",
+            "Is looking around camp for someone to share tongues with",
+            "Winked playfully at a Clanmate from across the clearing!",
+            "Has received several invitations to share tongues... and is deciding which to take",
+            "Is basking in a sunray in the camp clearing",
+            "Has the kits and apprentices very engaged in a very, very tall tale",
+            "Plans to go on a stroll with some cat today",
+            "Is swishing {PRONOUN/m_c/poss} tail back and forth in a laid back manner",
+            "Is grooming {PRONOUN/m_c/poss} already silky soft coat",
+            "Is purring warmly",
+            "Is purring loudly",
+            "Is purring sweetly",
+            "Is this moon's heartthrob",
+            "Has recently given a wonderful speech to fellow Clanmates, boosting morale",
+            "Spends all day chatting and laughing with some other warriors",
+            "Forgot where {PRONOUN/m_c/subject} put something, and every other warrior offers to help {PRONOUN/m_c/object} find it",
+            "Is a little exhausted from how many cats come to {PRONOUN/m_c/object} for things",
+            "Feels happy knowing {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} someone other's can talk to",
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be dazzling, but not too dazzling"
+        ],
+        "main_trait_constraint": [
+            "charismatic"
+        ]
+    },
+    {
+        "id": "childish_gen",
+        "thoughts": [
+            "Is chasing a butterfly around the camp",
+            "Is playing with {PRONOUN/m_c/poss} food",
+            "Is giggling about Clan drama with friends",
+            "Is crunching leaves with {PRONOUN/m_c/poss} paws",
+            "Is distracted by a shiny Twoleg trinket {PRONOUN/m_c/subject} found",
+            "Is sooooooo booorrredddd",
+            "Is gnawing on a small bone busily",
+            "Is batting at a ball of moss",
+            "Is preoccupied playing an important game of hide and seek",
+            "Is laughing gleefully!",
+            "Gets in an argument over something utterly trivial",
+            "Stuck {PRONOUN/m_c/poss} tongue out at a mouse that escaped {PRONOUN/m_c/poss} pounce",
```

Comment:
remove this one from here since it doesn't apply to medicine cat( apprentice)s and mediator( apprentice)s?

Reply:
moved to a better category